apps/meeting/src/components/DeviceSelection/CameraDevices/VideoTransformDropdown.tsx

```diff
@@ -55,7 +55,7 @@ export const VideoTransformDropdown: React.FC<Props> = ({
   ];
 
   // Creates a device based on the selections (None, Blur, Replacement) and uses it as input.
-  async function selectTransform(e: ChangeEvent<HTMLSelectElement>) {
+  const selectTransform = async (e: ChangeEvent<HTMLSelectElement>) => {
```

Comment:
What's the purpose for changing each of these? Is it for styling?

Reply:
Just for styling, no specific reason. We use arrow function most in the React SDK.